src/main/java/org/commcare/suite/model/QueryPrompt.java

```diff
@@ -24,6 +26,7 @@ public class QueryPrompt implements Externalizable {
     public static final String INPUT_TYPE_DATERANGE = "daterange";
     public static final String INPUT_TYPE_DATE = "date";
     public static final String INPUT_TYPE_ADDRESS = "address";
+    public static final String DEFAULT_VALIDATION_ERROR = "This answer is outside the allowed range";
```

Comment:
nit: more generic message (it may not be a range type, also user's don't think of these as 'answers'): 
```suggestion
    public static final String DEFAULT_VALIDATION_ERROR = "Input validation failed";
```